packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -122,6 +125,22 @@ export class ARRenderer extends EventDispatcher {
     super();
     this.threeRenderer = renderer.threeRenderer;
     this.threeRenderer.xr.enabled = true;
+
+    this.xrLight = new XREstimatedLight(this.threeRenderer);
+
+    this.xrLight.addEventListener('estimationstart', () => {
+      if (!this.isPresenting) {
+        return;
+      }
+
+      const scene = this.presentedScene!;
+      scene.add(this.xrLight!);
+
+      if (this.xrLight!.environment) {
+        this.oldEnvironment = scene.environment;
+        scene.environment = this.xrLight!.environment;
```

Comment:
Have you tested this with a shiny object? A big complaint we got in Scene Viewer is that shiny objects look matte because the ARCore environment image is so low-res (16x16 I believe) that it cannot produce a sharp reflection. I was thinking it would be better to either ignore its environment entirely or generate an LDR version of our PMREM and modulate it with the estimated environment (so that the high res one does most of the reflections while the low-res one takes care of the HDR lighting).